File: app/src/main/java.backup/com/lumiyaviewer/lumiya/voice/common/messages/VoiceEnableMic.java
/*
 * Decompiled with CFR 0.152.
 * 
 * Could not load the following classes:
 *  android.os.Bundle
 */
package com.lumiyaviewer.lumiya.voice.common.messages;

import android.os.Bundle;
import com.lumiyaviewer.lumiya.voice.common.VoicePluginMessage;

public class VoiceEnableMic
implements VoicePluginMessage {
    public final boolean enableMic;

    public VoiceEnableMic(Bundle bundle) {
        this.enableMic = bundle.getBoolean("enableMic");
    }

    public VoiceEnableMic(boolean bl) {
        this.enableMic = bl;
    }

    @Override
    public Bundle toBundle() {
        Bundle bundle = new Bundle();
        bundle.putBoolean("enableMic", this.enableMic);
        return bundle;
    }
}

File: Linkpoint/src/main/app_broken_backup/com/linkpoint/voice_backup/voice/common/messages/VoiceEnableMic.kt
/*
 * Decompiled with CFR 0.152.
 * 
 * Could not load the following classes:
 *  android.os.Bundle
 */
package com.linkpoint.voice.common.messages

import android.os.Bundle
import com.linkpoint.voice.common.VoicePluginMessage

class VoiceEnableMic
: VoicePluginMessage {
    Boolean enableMic

    VoiceEnableMic(Bundle bundle) {
        this.enableMic = bundle.getBoolean("enableMic")
    }

    VoiceEnableMic(Boolean bl) {
        this.enableMic = bl
    }

    @Override
    Bundle toBundle() {
        Bundle bundle = Bundle()
        bundle.putBoolean("enableMic", this.enableMic)
        return bundle
    }
}

